src/index.ts

```diff
@@ -80,27 +79,22 @@ export class MagicConnect extends Connector {
   private async isomorphicInitialize(): Promise<void> {
     if (this.eagerConnection) return
 
-    if (this.magic) {
-      const provider = new Web3Provider(this.magic.rpcProvider as any)
-      this.provider = new Eip1193Bridge(provider.getSigner(), provider)
-
-      // this.provider.on("connect", this.connectListener)
-      // this.provider.on("disconnect", this.disconnectListener)
-      // this.provider.on("chainChanged", this.chainChangedListener)
-      // this.provider.on("accountsChanged", this.accountsChangedListener)
+    if (this.provider) {
+      this.provider.on("connect", this.connectListener)
+      this.provider.on("disconnect", this.disconnectListener)
+      this.provider.on("chainChanged", this.chainChangedListener)
+      this.provider.on("accountsChanged", this.accountsChangedListener)
 
       this.eagerConnection = Promise.resolve()
     }
   }
 
-  /** {@inheritdoc Connector.connectEagerly} */
-  public async connectEagerly(): Promise<void> {
+  private async handleActivation(): Promise<void> {
```

Comment:
@ZYJLiu Why did this change from connectEagerly to handleActivation and change from public to private?